README.rst

```diff
@@ -40,14 +40,59 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
 
 ABOUT
 =====
 
-coala-bears is a Python package containing all the bears that is used by coala.
-
-Read more at our `documentation <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#guide-to-write-a-bear>`__.
+coala-bears is a Python package containing all the bears that are used by coala.
+It has been split from `coala <https://github.com/coala-analyzer/coala>`_.
```

Comment:
Do we need to mention the split ? That should be part of release notes, not readme ?